---
geometry_msgs/Pose[] poses
bool success
string message  # for error description